### PR TITLE
YAML file now gets packaged on build

### PR DESCRIPTION
**Pull request by hrshdhgd**
- [x] Restructured directory such that the schema file is within the `src/babelon` dir 
- [x] Updated via `poetry update`
- [x] Dev dependencies now in a group as per latest version of poetry prescription.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -119,8 +119,8 @@ cli_test:
 				--language-code de \
 				--field rdfs:label --field IAO:0000115 \
 				-o tests/tmp/example-augmented.babelon.tsv
-	#export OPENAI_API_KEY="" &&\
-	#babelon translate tests/tmp/example-augmented.babelon.tsv -o tests/tmp/example-translated.babelon.tsv
```
Comment (matentzn) [inline Makefile:123]: Why remove the two lines above?

File: pyproject.toml [hunk 2]
```diff
@@ -1,11 +1,10 @@
 [tool.poetry]
 name = "babelon"
-version = "0.2.1"
+version = "0.0.0"
```
Comment (matentzn) [inline pyproject.toml:3]: Do we have to change the version? We already have some published, I want to start at 0.2. if possible

**Review by matentzn — COMMENTED** [review]
@hrshdhgd feel free to merge you you looked at my comments. minimial version should be 0.2

File: Makefile (hunk 1 above)
Comment (hrshdhgd) [reply Makefile:123]: for some reason, `make cli_test` didn't like that line. It errors out stating the file path doesn't exist.

File: pyproject.toml (hunk 2 above)
Comment (hrshdhgd) [reply pyproject.toml:3]: No ....when you do a release via GitHub, it should get the version from the tag provided for release.

**Pull request merged**